### DISCUSSION TITLE
q-input textarea type expansion icon at bottom-right corner not well-formed in Chrome. It looks fine in Firefox and Edge.

### DISCUSSION THREAD

**Discussion by Ammamon** [quasarframework/quasar#16773] [General - Components / Directives / etc]
q-input textarea type expansion icon at bottom-right corner not well-formed in Chrome. It looks fine in Firefox and Edge.
Below is what shown in Chrom
![image](https://github.com/quasarframework/quasar/assets/7417777/e58a7000-d089-42f3-aa5a-67fd2c118f5a)
Below that of Firexfox
![image](https://github.com/quasarframework/quasar/assets/7417777/f8a28172-b58c-4d90-8fb0-e72ce5490e1d)

**Comment by pdanpdan** [comment]
Is it a requirement to have some activity from time to time (no matter how stupid) so that the account looks legit?
